### PR TITLE
Mrgreentea/feat/35 update calendar events

### LINKED ISSUE #35: Update events in calendar
We have the upcoming events for this year.  Update the data

### LINKED ISSUE #36: Link to event pages from calendar
Make sure to only do that if there is a page

### PR DESCRIPTION
**Pull request by MrGreenTea**
Closes #35 Closes #36

### CODE REVIEW

**Comment by MrGreenTea** [comment]
I haven't yet changed the data for the events. I first want to spend some thought about how to make the tests more resilient and not dependent on the current date.

Maybe we just use an environment variable `PLAYWRIGHT_TESTS` or something and then generate hard coded events?

File: src/components/Calendar.svelte [hunk 1]
```diff
@@ -15,27 +15,57 @@
...
+	const [firstEvent, secondEvent, thirdEvent] = events;
 </script>
 
 <div class="grid w-full grid-cols-4 lg:grid-cols-9 lg:px-20">
 	<div
 		class="col-start-1 col-end-5 row-end-2 lg:relative lg:col-end-6 lg:row-start-1 lg:row-end-4 lg:rounded-l-2xl lg:rounded-r-none"
 	>
 		<div class="relative h-[22rem] w-full lg:h-full">
-			{#each events as event, index}
+			{#if selectedEvent === 0}
```
Comment (MrGreenTea) [inline src/components/Calendar.svelte:27]: This should use `{#keyed}` instead of one if statement per event.

**Review by Han2-Ro — APPROVED** [review]
Cool, keep those events up to date

File: src/components/Calendar.svelte [hunk 2]
```diff
@@ -8,30 +10,34 @@
...
+						easing: cubicOut,
+					}}
 					loading="lazy"
-					id="event-{index}"
-					{...event.image.src.attributes}
-					src={event.image.src.src}
+					id={`event-${selectedEvent + 1}`}
+					{...currentEvent.image.src.attributes}
+					src={currentEvent.image.src.src}
 					alt="Leute sitzen am Tisch"
```
[the remark this answers is not in the document]
Comment (MrGreenTea) [reply src/components/Calendar.svelte:37]: See #65

**Pull request merged**